docs/reference/search/search-your-data/paginate-search-results.asciidoc

```diff
@@ -71,9 +71,9 @@ To get the first page of results, submit a search request with a `sort`
 argument. If using a PIT, specify the PIT ID in the `pit.id` parameter and omit
 the target data stream or index from the request path.
 
-IMPORTANT: We recommend you include a tiebreaker field in your `sort`. This
-tiebreaker field should contain a unique value for each document. If you don't
-include a tiebreaker field, your paged results could miss or duplicate hits.
```

Comment:
What if a user doesn't use PIT in a sort request?  Should we still keep this recommendation of adding a tie-breaking field to sort fields? 
Or is our official recommendation always use PIT with sort?